app/src/org/commcare/android/tasks/ZipTask.java

```diff
@@ -178,9 +165,8 @@ private boolean zipTargetFolder(File targetFilePath, String zipFile) throws IOEx
         ZipOutputStream out = new ZipOutputStream(new BufferedOutputStream(new FileOutputStream(zipFile)));
         
         try{
-        
             if(!targetFilePath.isDirectory()){
-                System.out.println("827: target was not folder, bad");

```

Comment:
Okay to take these print statements out? The 827 made them look like debug prints.